### PR TITLE
Turn on tests for WASM and add instructions for running.

### PR DESCRIPTION
**Pull request by tylerhou**
Depends on #414

### CODE REVIEW

**Comment by shadaj** [comment]
Nice! We should definitely run these tests in CI...

File: hydroflow/wasm.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+Install `wasm-bindgen-cli` globally: `cargo install wasm-bindgen-cli`.
+`wasm-bindgen-cli` provides a test runner harness.
+
+### Running Tests
+
+Run (in the root hydroflow directory):
+
+```
+CARGO_TARGET_WASM32_UNKNOWN_UNKNOWN_RUNNER=wasm-bindgen-test-runner cargo test --target wasm32-unknown-unknown -p hydroflow --tests
```
Comment (MingweiSamuel) [inline hydroflow/wasm.md:15]: Just add this to ci.yml I guess?

**Review by MingweiSamuel — APPROVED** [review]
Would be good to have in CI and would be nice if it was opt-out instead of opt-in

File: hydroflow/wasm.md (hunk 1 above)
Comment (tylerhou) [reply hydroflow/wasm.md:15]: Talked to Shadaj, will do in followup.

**Comment by tylerhou** [comment]
Added `multiplatform_test` proc macro create and added internal tests for that crate.

File: hydroflow/wasm.md (hunk 1 above)
Comment (tylerhou) [reply hydroflow/wasm.md:15]: #428

**Review by shadaj — APPROVED** [review]
LGTM!

**Pull request merged**